docs/source/reference/cloud-administration.rst

```diff
@@ -0,0 +1,50 @@
+.. _cloud-administration:
+
+Cloud Administration
+=============================
+
+SkyPilot is designed to minimize cloud administration and setup steps. It should work out-of-the-box for most users without any additional setup.
+
+For users who want to use advanced cloud-specific features (e.g., use SSO; have fine-grained control over permissions; etc.), this page provides instructions to do so.
+
+.. tip::
+
+   More topics will be added to this page gradually as they surface.
+
+   Contributions are especially welcome on administration as cloud accounts differ case-by-case.
+
+
+AWS
+-------------------------------
+
+.. _aws-sso:
+
+AWS SSO
+~~~~~~~~~~~
+`AWS IAM Identity Center <https://aws.amazon.com/iam/identity-center/>`_ (Successor to AWS Single Sign-On, or SSO) is supported.
+
+To use it, ensure that your machine `has AWS CLI V2 installed <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_ (by default, ``pip install skypilot[aws]`` installs V1; V2 cannot be installed via pip).
+
+You can use the following to check version:
+
+.. code-block:: console
+
+    $ aws --version
+
+
+Then, after the usual ``aws configure sso`` and ``aws sso login --profile <profile_name>`` commands, SkyPilot will work as usual.
```

Comment:
Are we sure that SkyPilot will work with v2 as the default `aws` command?

Currently, what I did for SSO was to create an `awsv2` command that is only used for `awsv2 configure sso` and `awsv2 sso login`, while let the SkyPilot uses the `v1`.

Reply:
Working like a champ for me. I admit I'm using a subset of features though. Is there a list of CLI-based commands that Sky runs?

Reply:
Tested all of the above with `aws` aliased to v2.